person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   private
```

Comment:
- Hey, :wave: Also, it is great and more of a good convention in ruby to have all your public methods in a class come first before your private method. it keeps your code readable. instead of denoting the `public` to indicate a method. every method in a class is public except denoted with the `private` key. Instead of having to add the `public` kindly move 
```ruby
  private


  def of_age?
    @age >= 18
  end
```
to the bottom just after the `correct_name` method, then remove the `public` as you've added.